components/Common/Tabs/index.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+import type { FC } from 'react';
+
+import { Tabs, TabsList, TabsTrigger, TabsContent } from './index';
+
+const DecoratedTabs: FC = () => (
```

Comment:
Instead of doing this, can you simply pass Tabs as the component, and pass all these values as the `children` arg within the `export const Default`?